src/media/css/games/games.styl

```diff
@@ -0,0 +1,191 @@
+@import '../lib';
+
+$color--games-red-light = #C93333;
+$color--games-red-dark= #410000;
+$color--games-yellow-light = #FFB300;
+$color--games-yellow-dark = #573D00;
+$color--games-blue-light = #0092FF;
+$color--games-blue-dark = #001D48;
+$color--games-green-light = #009900;
+$color--games-green-dark = #003600;
+$color--games-pink-light = #CC3388;
+$color--games-pink-dark = #541538;
+$color--games-indigo-light = #197B99;
+$color--games-indigo-dark = #541538;
+$color--games-orange-light = #CC440D;
+$color--games-orange-dark = #511B06;
+$color--games-purple-light = #7A3675;
+$color--games-purple-dark = #251023;
+
+
+games-gradient($from, $to) {
+    background: linear-gradient(to bottom, $from, $to);
+}
+
+games--listing() {
+    .games-listing & {
+        {block}
+    }
+}
+
+game--hover() {
+    .game:hover & {
+        {block}
+    }
+}
+
+
+[data-page-type~="games"] {
+    #page {
+        padding-top: 0;
+    }
+}
+
+.games-title {
+    align-content: center;
+    background: #006DBF;
+    display: flex;
+    justify-content: center;
+    padding: 20px 0;
+    width: 100%;
+
+    h1 {
+        color: $greyscale-white;
+        font-size: 30px;
+        font-weight: 600;
+    }
+}
+
+.game {
+    align-items: center;
+    background-size: cover;
+    display: flex;
+    justify-content: center;
+    height: 250px;
+    position: relative;
+
+    &:nth-child(8n) .game-gradient {
+        games-gradient($color--games-red-light, $color--games-red-dark);
+    }
+    &:nth-child(8n + 1) .game-gradient {
+        games-gradient($color--games-yellow-light, $color--games-yellow-dark);
+    }
+    &:nth-child(8n + 2) .game-gradient {
+        games-gradient($color--games-blue-light, $color--games-blue-dark);
+    }
+    &:nth-child(8n + 3) .game-gradient {
+        games-gradient($color--games-green-light, $color--games-green-dark);
+    }
+    &:nth-child(8n + 4) .game-gradient {
+        games-gradient($color--games-pink-light, $color--games-pink-dark);
+    }
+    &:nth-child(8n + 5) .game-gradient {
+        games-gradient($color--games-indigo-light, $color--games-indigo-dark);
+    }
+    &:nth-child(8n + 6) .game-gradient {
+        games-gradient($color--games-orange-light, $color--games-orange-dark);
+    }
+    &:nth-child(8n + 7) .game-gradient {
+        games-gradient($color--games-purple-light, $color--games-purple-dark);
+    }
+}
+
+.game-gradient,
+.game-overlay {
+    height: 100%;
+    left: 0;
+    position: absolute;
+    top: 0;
+    width: 100%;
+}
+
+.game-gradient {
```

Comment:
It would be really nice if we could do this as `.game::after` and avoid the empty `<div>` completely.